warehouse/warehouse.go

```diff
@@ -732,15 +641,139 @@ func (wh *HandleT) mainLoop() {
 	}
 }
 
+func (wh *HandleT) getUploadsToProcess(availableWorkers int, skipIdentifiers []string) ([]*UploadJobT, error) {
+
+	var skipIdentifiersSQL string
+	if len(skipIdentifiers) > 0 {
+		skipIdentifiersSQL = fmt.Sprintf(`and concat(t.namespace, ':', t.destination_id) != ALL($1)`)
```

Comment:
This concat should be with _ instead of column, because that's what we get from workerIdentifier func